Acrolinx/AcrolinxURLProvider.py

```diff
@@ -23,24 +23,26 @@
 
 __all__ = ["AcrolinxURLProvider"]
 
-URL = "https://{}:{}@download.acrolinx.com:1443/api/deliverablePackages/575b1d0d401ae30b00e90f40/download/latest?preserve_credentials=true&proxy=true"
+URL = "https://{}:{}@download.acrolinx.com:1443/api/deliverables/{}/download/latest"
 
 
 class AcrolinxURLProvider(URLGetter):
     """Provides a download URL for Acrolinx."""
 
     description = __doc__
     input_variables = {
+        "acronlix_uuid": {"required": True, "description": "UUID that corresponds to a specific Acrolinx customer portal"},
```

Comment:
I am actually not sure if this corresponds to different customers or what causes this UUID to change 🤔 @nmcspadden can you check if your portal still has this same static UUID? You can check via Chrome Developer Tools -> Console -> Network while performing a download in the GUI